common/src/main/resources/Operator.properties

```diff
@@ -178,6 +178,7 @@ WLSKO-0218=Domain custom resource converter: Name of the converted file. If not
   file name by appending "__converted." and the input file extension to the base name of the input file name.
 WLSKO-0219=Enable overwriting the existing output file, if any.
 WLSKO-0220=Successfully generated upgraded domain custom resource file ''{0}''.
+WLSKO-0221=WebLogic Domain custom resource conversion webhook failed due to ''{0}''.
```

Comment:
A) Is there additional information available? Name of the domain? (anything to help identify the source when available)

B) I still see missing decorations on other WH text ^^^, for example 0211, 0206, 0207, and possibly others

Reply:
(a) I added conversion request details to the message. 
(b) Since 0211, 0206 and 0207 were added in previous PR, I missed it earlier. I have added decoration now.